### ISSUE TITLE
Create a github actions that run through the workflow every time you do a pull request

### ISSUE THREAD

**Issue by scorreard** [bcgsc/Canadian_Biogenome_Project#3]

**Comment by oneillkza** [comment]
I think this would be a good idea. I would try it with a small example, though. Maybe select reads from just one region of an assembly (both PacBio and Hi-C) to use as input for that.

**Comment by scorreard** [comment]
Thanks Kieran. There is already minimal test dataset, it is always necessary with nextflow pipeline :)

The problem is more the GitHub test, they have limited software / memory / cpus

**Comment by scorreard** [comment]
Indeed, the free version of GitHub Actions do not allow to process such pipeline (too low memory / CPUs)
Pipeline get cancelled by GitHub action

<img width="637" alt="image" src="https://github.com/bcgsc/Canadian_Biogenome_Project/assets/54953390/33372876-4b08-45b1-9dcd-1b01bc5a9b3e">
